Library/Homebrew/test/utils/git_spec.rb

```diff
@@ -2,8 +2,10 @@
 
 require "utils/git"
 
-describe Git do
+describe Utils::Git do
   before do
+    described_class.clear_available_cache
```

Comment:
@reitermarkus I think this change or similar may be breaking parallel tests. Seeing lots of exceptions on macOS and Linux (that go away on rerun) when they can't find Git: https://github.com/Homebrew/brew/runs/1025893539?check_suite_focus=true#step:13:129

Reply:
Fixed in https://github.com/Homebrew/brew/pull/8486.